assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -403,6 +509,80 @@ const baseSelectors = {
 		return modules[ slug ];
 	} ),
 
+	/**
+	 * Gets the display mode for module's settings.
+	 *
+	 * Returns one of: "closed", "view", "edit", or "locked".
```

Comment:
Per this PR, can also be "saving". Let's rely on constants though, as mentioned in my other comment.

Reply:
Updated.